rdt/transformers/numerical.py

```diff
@@ -68,11 +68,12 @@ def __init__(self, dtype=None, nan='mean', null_column=None, rounding=None,
         self.rounding = rounding
         self.min_value = min_value
         self.max_value = max_value
+        self.null_transformer = NullTransformer(self.nan, self.null_column, copy=True)
```

Comment:
I do not think this change should be done, since we will want to re-create the NullTransformer instance if the `fit` method is called more than once.